css/index.css

```diff
@@ -51,7 +46,9 @@ table {
 * {
     box-sizing: border-box;
 }
-
+html { 
+    font-size: 62.5%;
```

Comment:
Important to note that this is what allows font sizes to be changed to rem in base of 10.